combat-trainer.lic

```diff
@@ -1175,7 +1175,17 @@ class TrainerProcess
     when 'App Careful'
       appraise(game_state, 'careful')
     when 'Tactics'
-      bput(%w(weave bob circle).sample, 'roundtime', 'There is nothing else', 'Face what', 'You must be closer', 'You must be standing', 'Strangely, you don\'t feel like fighting right now', 'flying too high for you to attack') unless game_state.npcs.empty?
+      if not game_state.npcs.empty?
```

Comment:
I recommend moving this logic into a helper method. Then instead of an `if` around all of the logic, you can use a guard clause (see line 1295 for an example).